lib/alice/bot/supervisor.ex

```diff
@@ -1,14 +1,15 @@
 defmodule Alice.Bot.Supervisor do
-  @moduledoc false
+  @moduledoc """
+  Supervises any Alice.Bot process that are started.
+  """
 
   use Supervisor
 
-  def start_link(opts \\ []) do
-    Supervisor.start_link(__MODULE__, :ok, opts)
+  def start_link(_) do
+    Supervisor.start_link([
+      worker(Alice.Bot, [], restart: :transient)
+    ], strategy: :simple_one_for_one, name: __MODULE__)
```

Comment:
Same comment here about going with a module based supervisor. I also don't think we need a `simple_one_for_one` supervisor here. We may have discussed why a straight `one_to_one` wouldn't work, but i don't remember why that is.

Reply:
I left it SOFO because the bot process is started by the user's bot project after the entire Alice tree starts. Using SOFO, we don't have to double-supervise the bot process.

Reply:
However, having said that, I'm not 100% happy with having to start the bot there kind of manually, so maybe that's something we still have to figure out.